### DISCUSSION TITLE
Find albums in which a photo is in

### DISCUSSION THREAD

**Discussion by daniambrosio** [RhetTbull/osxphotos#412] [Q&A]
I was looking for a way to find in which albums a photo is in. I thought I could use osxphotos, but I noticed that there is no query command to search for a specific filename. There are albums, folders, titles... but no query command for filename. 

Is that true or am I missing something? 

Cheers,

**Comment by RhetTbull** [comment]
There wasn't a way to do this...but I just added it in [v0.41.8](https://github.com/RhetTbull/osxphotos/releases/tag/v0.41.8).  With this version you can use `--name FILENAME` to search a specific filename.

**Reply by daniambrosio** [reply]
Wow!! thanks that is really useful!
